internal/bed/bed.go

```diff
@@ -0,0 +1,58 @@
+package bed
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// Note that the first column is 1 when user sets column locations,
+// but that .VerifyAndHandle() will correct this to standard indexes
+type Bedfile struct {
+	Input     string   `env:"INPUT_FILE" required:"" short:"i" help:"Bed file path"`
+	Output    string   `env:"OUTPUT_FILE" short:"o" help:"Path for the output file. If unset the output will be written to stdout"`
+	StrandCol int      `env:"STRAND_COL" help:"The column containing the strand information (first column is 1)"`
+	FeatCol   int      `env:"FEAT_COL" help:"The column containing the feature information (first column is 1)"`
+	Header    []string `kong:"-"`
+	Lines     []Line   `kong:"-"`
+}
+
+type Line struct {
+	Chr    string
+	Start  int
+	Stop   int
+	Strand string
+	Feat   string
+	Full   []string
+}
+
+// Bed file constants
+const (
+	chrIdx   = 0
+	startIdx = 1
+	stopIdx  = 2
+)
+
+// Verifies the user input for Bedfile
+// fixes path and subtracts 1 from cols to be able to use normal indexing
+func (bf *Bedfile) VerifyAndHandle() error {
+	if bf.StrandCol != 0 {
+		if bf.StrandCol < stopIdx+1 && bf.StrandCol != 0 {
```

Comment:
In this case we actually need to check both because of the input.
So we are checking for two things:
1. That the user has not input a column that is less than 3 (because those are taken by chr, start and stop). If this is the case we want to give an error.
2. We check that Strand is not 0, as this is the default empty value. If this is the case we do not want to give an error as it could be because the user just never set the strand.  

We also do these same check for the feature column.

Maybe a better solution would be to set the default to for example -9999, to make it more obvious that this means that the strand or feature column was not set by the user?